install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -112,6 +112,10 @@ spec:
 {{- end }}
           - name: PILOT_DISABLE_XDS_MARSHALING_TO_ANY
             value: "1"
+{{- if .Values.proxyProtocol }}
+          - name: PROXY_PROTOCOL_ENABLED
```

Comment:
I would really like if the values key matches  the env variable. No need to have 2 different names.

Reply:
And I really hate that the name of the protocol is "the proxy protocol". Is there any other name we can use ? An RFC or something ? There are many proxy protocols - Envoy is a proxy for http and TCP, we want socks support, etc.

Reply:
Funny, I can't find an RFC, but I do see some definitions from the squid (https://github.com/squid-cache/squid/blob/master/doc/rfc/proxy-protocol.txt) and haproxy (https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt).

The squid one seems to be more recent, but that's besides your point.  We can name the setting unmask_client_address, or PILOT_PROXY_CLIENT_REAL_ADDRESS or something like that?